regulations/static/regulations/js/source/views/preamble-view.js

```diff
@@ -17,12 +17,19 @@ var PreambleView = ChildView.extend({
     'click .activate-read': 'read'
   },
 
-  initialize: function() {
+  initialize: function(options) {
+    this.options = options;
+    this.id = options.id;
+    this.url = 'preamble/' + this.id.split('-').join('/');
```

Comment:
Based on the Backbone docs and other views, no--we pass the prefix as `root` to the router in `router.js`. We do need to include the prefix when making AJAX requests, since `MetaModel` doesn't automatically add it in `request`--although it could.